packages/analytics-browser/src/browser-client.ts

```diff
@@ -112,7 +112,9 @@ export class AmplitudeBrowser extends AmplitudeCore implements BrowserClient, An
 
     let remoteConfigClient: IRemoteConfigClient | undefined;
     // Create remote config client and subscribe to analytics configs
-    if (browserOptions.remoteConfig?.fetchRemoteConfig) {
+    /* istanbul ignore next */
+    const fetchRemoteConfig = browserOptions.remoteConfig && browserOptions.remoteConfig.fetchRemoteConfig;
+    if (fetchRemoteConfig) {
```

Comment:
If we are using both of them, we have to keep them synced. I just added one line to the newly added test and it will fail.
<img width="1504" height="1007" alt="Image" src="https://github.com/user-attachments/assets/7820bbc6-9f76-47db-9a14-54eb6c87fe6d" />

We can just keep the original code and make sure we update the deprecated one based on the new one